src/App.tsx

```diff
@@ -15,47 +15,111 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  NONE,
+  ALPHABET,
+  LENGTH,
+}
```

Comment:
enum keys should be capitalized, it's a common practice
```suggestion
enum SortType {
  None,
  Alphabet,
  Length,
}
```